src/cm_common/cm_stat.h

```diff
@@ -326,6 +326,10 @@ extern "C"
     unsigned int bt_num_splits;
     unsigned int bt_num_merges;
     unsigned int bt_num_get_stats;
+    unsigned int bt_copy_leaf_nolatch_success_cnt;
+    unsigned int bt_copy_leaf_nolatch_fails_cnt;
+    unsigned int bt_acquire_tran_bcb_area_success_cnt;
+    unsigned int bt_acquire_tran_bcb_area_fails_cnt;
```

Comment:
Is not about memory allocation. Currently, we use only one bcb_area for a transaction. If we have a join, for instance, the page from first table will be copied without the latch, while the page from the second page will be S-latched. Based on this statistics, we can think about adding more bcb areas for a transaction (copy several pages without S-latch, when needs at least two pages at the same time). If you think that is not important, I will remove this statistic.